k-distribution/include/builtin/domains.k

```diff
@@ -586,20 +598,16 @@ module STRING-BUFFER-SYNTAX
   syntax String ::= StringBuffer2String ( StringBuffer ) [function, functional, hook(BUFFER.toString)]
 endmodule
 
-module STRING-BUFFER-SYMBOLIC [symbolic]
-  imports STRING-BUFFER-SYNTAX
-  imports STRING-SYNTAX
-  imports BOOL-SYNTAX
-
-  syntax StringBuffer ::= String
-
-  rule .StringBuffer => ""
+module STRING-BUFFER-CONCRETE [concrete]
```

Comment:
this is functionally correct but these modules now seem redundant to me. We should be able to do it with 2 modules and not 4.

Reply:
Is this what you meant?